testsuite/tests/typing-local/local.ml

```diff
@@ -320,15 +320,57 @@ val apply2 : int -> local_ 'a -> int -> int = <fun>
 |}]
 let apply3 x = g x x x
 [%%expect{|
-Line 1, characters 15-22:
+Line 1, characters 15-20:
 1 | let apply3 x = g x x x
-                   ^^^^^^^
+                   ^^^^^
+Error: This application is complete, but further arguments were provided afterwards.
+       With local arguments or closures, these are not allowed in the same application.
+  Hint: Try wrapping the marked application in parentheses.
```

Comment:
Nitpick: I honestly read "these" as referring first to "closures", which didn't make sense, and then "local arguments or closures", which still didn't make sense, and then I finally found "further arguments". With that in mind, what do you think of:

```suggestion
Error: This application is complete, but further arguments were provided afterwards.
       When passing local values, extra arguments must appear in a separate application.
  Hint: Try wrapping the marked application in parentheses.
```

Reply:
```
Error: This application is complete, but further arguments were provided afterwards.
       When passing or calling a local value, extra arguments are passed in a separate application.
  Hint: Try wrapping the marked application in parentheses.
```

?